README.md

```diff
@@ -0,0 +1,152 @@
+
+Let *worldReadable* be true if you would like this graph to be accessible to anyone with a link to the plot.  If this is false, the graph will only be accessible to your account by viewing the plots you own.
+
+Let *traces* be a list of the data point names you would like to graph.  Each Plotly graph can display many concurrent values known as traces, but you must list them all here before plotting them.
+
+```squirrel
+local callback = function(response, plot){
+    server.log(response.body);
+    server.log("See plot at " + plot.getUrl());
+}
+myPlot <- Plot("<YOUR_USERNAME>", "<YOUR_API_KEY>", "weather_data", true, ["temperature", "inside_humidity", "outside_humidity"], callback);
+```
+
+## Class Methods
+
+## plot.getUrl()
+
+Returns a string with the URL of the graph that this object generates.  Note that if *worldReadable* was set to false in the constructor, this link will only be viewable when logged into Plotly.
+
+```squirrel
+local plotUrl = myPlot.getUrl();
+```
+
+## plot.setTitle(*title [, callback]*)
+
+Sets the title that will be displayed on this graph.
+
+```squirrel
+myPlot.setTitle("Weather at Station 7");
+```
+
+## plot.setAxisTitles(*xAxisTitle, yAxisTitle*)
+
+Sets the labels that will be applied to the standard x- and y-axes on this graph.  If either argument is null or empty, that axis title will not be changed.
+
+```squirrel
+myPlot.setAxisTitles("Time", "Temperature (°F)");
+```
+
+## plot.addSecondYAxis(*axisTitle, traces [, callback]*)
+
+Adds a second y-axis on the right side of the graph and assigns the specified traces to it.  *traces* should be a list of the string names of traces as passed in the *traces* argument to the constructor.
+
+```squirrel
+myPlot.addSecondYAxis("Humidity (%)", ["inside_humidity", "outside_humidity"]);
+```
+
+## plot.setStyleDirectly(*styleTable [, callback]*)
+
+Sets the style of the graph by passing a description directly to the Plotly API.  This allows for advanced styling options that this library does not have specific methods for.
+
+*styleTable* should be a Squirrel list or table that will be parsed into JSON.  See the [Plotly API docs] (https://plot.ly/rest/) for details on how to format this argument.
+
+Note that there are several caveats to using this method:
+
+- This will entirely overwrite style parameters previously set using methods like `AddSecondAxis` or `setStyleDirectly`.
+- If there is an error in formatting *styleTable*, an error may be printed to the console or the call may silently fail.
+
+```squirrel
+local style =
+[
+    {
+        "name" : "temperature",
+        "type": "scatter",
+        "marker": {"symbol": "square", "color": "purple"}
+    },
+    {
+        "name" : "inside_humidity",
+        "type": "scatter",
+        "marker": {"symbol": "circle", "color": "red"}
+    }
+];
+myPlot.setStyleDirectly(style);
+```
+
+## Plotly.setLayoutDirectly(*layoutTable [,callback]*)
+
+See documentation for `setStyleDirectly`.
```

Comment:
How do I use setStyleDirectly documentation for this request (need a bit more information)